src/databricks/labs/ucx/source_code/directfs_access.py

```diff
@@ -54,6 +54,11 @@ def from_dict(cls, data: dict[str, Any]) -> Self:
     assessment_start_timestamp: datetime = datetime.fromtimestamp(0)
     assessment_end_timestamp: datetime = datetime.fromtimestamp(0)
 
+    def as_dict(self):
+        result = dataclasses.asdict(self)
+        result["source_lineage"] = self.source_lineage
```

Comment:
```suggestion
        result["source_lineage"] = [dataclasses.asdict(_) for _ in self.source_lineage]
```

.. or even `_.as_dict()`


otherwise it's not a dictionary all along. what's the purpose of this method?

Reply:
it was precisely to avoid that, but i's no longer used. removed